CHANGELOG.md

```diff
@@ -733,3 +733,20 @@ should use 4.0.1-alpha.0 for testing.
 -   Dependency tree cannot be resolved by Yarn due to old deprecated packages picked by yarn - fixed (#5382)
 
 ## [Unreleased]
+
```

Comment:
I think at some point we'll want to write a script that parses all the `CHANGELOG.md`s in each package and updates this file so it doesn't have to be done manually each time

Reply:
Opened an issue for this suggestion: https://github.com/web3/web3.js/issues/5451